src/main/kotlin/org/rust/ide/intentions/AddCurlyBracesIntention.kt

```diff
@@ -0,0 +1,76 @@
+package org.rust.ide.intentions;
+
+import com.intellij.codeInsight.intention.PsiElementBaseIntentionAction
+import com.intellij.openapi.editor.Editor
+import com.intellij.openapi.project.Project
+import com.intellij.psi.PsiElement
+import org.rust.lang.core.psi.RustElementFactory
+import org.rust.lang.core.psi.RustUseItemElement
+import org.rust.lang.core.psi.util.parentOfType
+
+/**
+ * Adds curly braces to singleton imports, changing from this
+ *
+ * ```
+ * import std::mem;
+ * ```
+ *
+ * to this:
+ *
+ * ```
+ * import std::{mem};
+ * ```
+ */
+class AddCurlyBracesIntention : PsiElementBaseIntentionAction() {
+    override fun getText() = "Add curly braces"
+    override fun getFamilyName() = getText()
+    override fun startInWriteAction() = true
+
+    override fun invoke(project: Project, editor: Editor?, element: PsiElement) {
```

Comment:
You should be able to safely remove `?` from the `Editor?` here.

https://youtrack.jetbrains.com/issue/KT-12213
https://youtrack.jetbrains.com/issue/KT-12381